### PR TITLE
[P9] - Basic player prompt animation for moveable tiles in Steven scene.

### PR DESCRIPTION
**Pull request by EpicPants90**
P9 Tiles you can move to glow green now. Basic but ok for now.

### CODE REVIEW

**Comment by EpicPants90** [comment]
https://trello.com/c/h5Mh3u5o/22-p9-animation-player-prompts-flashing

**Comment by EpicPants90** [comment]
Bug with missing animator should be fixed now.

**Comment by alasdairhurst** [comment]
@EpicPants90 can you make it so it throbs just when the player is picked up pleeees? :+1:

File: Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/BoardUI.cs [hunk 1]
```diff
@@ -1,8 +1,43 @@
...
+            //if already animating then reset it
+            if (nextAnimator.GetTime() > 0)
+                nextAnimator.SetTime(0);
+
+            nextAnimator.SetBool("CanBeMovedTo", true);
+            _animatingTiles.Add(nextAnimator);
+        }
+    }
+
+    public void PlayerPrompt_DefualtTiles()
```
Comment (ewanharris) [inline Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/BoardUI.cs]: Just noticed the misspelling, it's up to you whether you want to fix now or follow on later

File: Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/OverworldUI.cs [hunk 2]
```diff
@@ -10,15 +10,23 @@ public class OverworldUI : MonoBehaviour
...
         _CommanderUI.OnCommanderMoved += _CommanderUI_OnCommanderMoved;
         _CommanderUI.OnStartDrag += _CommanderUI_OnStartDrag;
 		_CommanderUI.OnCommanderDrop += _CommanderUI_OnCommanderDrop;
 		_CommanderUI.OnCommanderGrounded += _CommanderUI_Grounded;
+        _CommanderUI.OnDropCommander += _CommanderUI_OnDropCommander;
+    }
+
+    void _CommanderUI_OnDropCommander(TileData tile)
+    {
+        _BoardUI.PlayerPrompt_DefualtTiles();
```
Comment (ewanharris) [inline Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/OverworldUI.cs]: Ditto here

**Comment by ewanharris** [comment]
Merging! :+1:

**Pull request merged**